src/clients/VestingData.ts

```diff
@@ -172,26 +182,125 @@ async function getVestingContractDataV2(
   const token = getTokenSymbolFromAddress(tokenContractAddress)
 
   return {
+    cliff: toISOString(contractCliff),
+    vestedPerPeriod: vestedPerPeriod,
     ...getVestingDates(contractStart, contractEndsTimestamp),
+    vested: released + releasable,
     released,
     releasable,
     total,
+    token,
     status,
     start_at,
     finish_at,
-    token,
-    vested: released + releasable,
   }
 }
 
-export async function getVestingWithLogs(
-  vestingAddress: string | null | undefined,
-  proposalId?: string
-): Promise<VestingWithLogs> {
-  if (!vestingAddress || vestingAddress.length === 0) {
-    throw new Error('Unable to fetch vesting data for empty contract address')
+export function parseVestingData(vestingData: SubgraphVesting): Vesting {
```

Comment:
We could move the parsing functions to another file like `utils/vesting.ts` as this file was focused on querying Vesting contract data directly from the contracts.

Reply:
Or move it to the VestingService file as it is not used anywhere else.